apps/judicial-system/api/src/app/modules/auth/auth.service.ts

```diff
@@ -23,7 +23,7 @@ export class AuthService {
     return await res.json()
   }
 
-  validateUser(user?: User): boolean {
-    return Boolean(user?.active)
+  validateUser(user: User): boolean {
+    return Boolean(user.active)
```

Comment:
do we still need the boolean here?

Reply:
It is not necessary for correctness, but using it is one way to deal with a "strict typescript" warning we get because user can be undefined.